### ISSUE TITLE
paftools call empty output

### ISSUE THREAD

**Issue by malmarri** [lh3/minimap2#361]
Hi,

Using 2.11-r797

I ran `paftools stat ` on a paf file and correctly found the statistics:

Number of mapped sequences: 6507621
Number of primary alignments: 6507847
Number of secondary alignments: 913944
Number of primary alignments with >65535 CIGAR operations: 0
Number of bases in mapped sequences: 3714419261
Number of mapped bases: 3709199992
Number of insertions in [0,50): 241810
Number of insertions in [50,100): 388
Number of insertions in [100,300): 134
Number of insertions in [300,400): 12
Number of insertions in [400,1000): 46
Number of insertions in [1000,inf): 7
Number of deletions in [0,50): 280118
Number of deletions in [50,100): 638
Number of deletions in [100,300): 242
Number of deletions in [300,400): 228
Number of deletions in [400,1000): 50
Number of deletions in [1000,inf): 21

However when I try to call using` paftools call` (on the same _sorted_ paf file) I find 0 variants:

0 reference bases covered by exactly one contig
0 substitutions; ts/tv = NaN
0 1bp deletions
0 1bp insertions
0 2bp deletions
0 2bp insertions
0 [3,50) deletions
0 [3,50) insertions
0 [50,1000) deletions
0 [50,1000) insertions
0 >=1000 deletions
0 >=1000 insertions

**Comment by lh3** [comment]
paftools.js call is designed for good assemblies. Your assembly is too fragmented. You need to change option `-L` and `-l`.

**Comment by clairemerot** [comment]
Hello,
I'm reopening this issue as I am facing the same problem.
I have aligned a fragmented genome (contigs) to a chromosome-scale reference with minimap2
This is the head of the delta file
contig_10001    118359  14200   19610   -       Chr09   65625476        51116011        51121424        5385    5426    1       NM:i:41 ms:i:4787       AS:i:4787       nn:i:0   tp:A:P  cm:i:112        s1:i:1378       s2:i:1395       de:f:0.0072     rl:i:29276      cg:Z:554M1D127M1D69M1I144M1I164M1I193M1D282M1D61M1D219M1I274M1D55M1I147M1I210M1I85M1D12M1D75M1D223M1D635M1D49M1D196M1I239M1I239M1I104M1I33M1D183M1D58M2D27M2I740M

When I run
k8 $paftools stat $INPUT_PAF
I get interesting results:

Number of mapped sequences: 7832
Number of primary alignments: 140677
Number of secondary alignments: 190645
Number of primary alignments with >65535 CIGAR operations: 0
Number of bases in mapped sequences: 2758757016
Number of mapped bases: 2465771159
Number of insertions in [0,50): 2929678
Number of insertions in [50,100): 47281
Number of insertions in [100,300): 47040
Number of insertions in [300,400): 8158
Number of insertions in [400,1000): 11935
Number of insertions in [1000,inf): 13607
Number of deletions in [0,50): 2493451
Number of deletions in [50,100): 42098
Number of deletions in [100,300): 43401
Number of deletions in [300,400): 7390
Number of deletions in [400,1000): 10780
Number of deletions in [1000,inf): 12375

but when I run 
k8 $paftools call $INPUT_PAF -f $REF -L 50 -l 10
I get nothing

R       Chr28   13024225        13119856
998480065 reference bases covered by exactly one contig
0 substitutions; ts/tv = NaN
0 1bp deletions
0 1bp insertions
0 2bp deletions
0 2bp insertions
0 [3,50) deletions
0 [3,50) insertions
0 [50,1000) deletions
0 [50,1000) insertions
0 >=1000 deletions
0 >=1000 insertions

I have tried playing with the L and l options but none of them worked. I have tried with and without giving the -f genome.fasta, still nothing. I have also tried using a paf file from other program (winnowmap) without success.

Any idea what am I doing wrong? @malmarri have you been able to solve this problem on your dataset?
Thanks to all of you for your help
Best regards
Claire
